styles/languages/c.less

```diff
@@ -3,13 +3,16 @@
     color: @red;
   }
   .keyword.control.import {
-    color: @red;
+    color: @orange;
+  }
+  .punctuation.definition.keyword {
+    color: @orange;
```

Comment:
`keyword.control.import` has been renamed to `keyword.control.directive`
`punctuation.definition` is now part of `keyword` so it can be removed
```less
.keyword.control.directive {
  color: @orange;
}
```

Refs atom/language-c#102